internal/controller/dependencyupdatecheck_controller.go

```diff
@@ -188,14 +188,15 @@ func (r *DependencyUpdateCheckReconciler) createPipelineRun(name string, comp co
 	if err != nil {
 		return nil, err
 	}
+	renovateJsConfig := "module.exports = " + renovateConfig
```

Comment:
Is this necessary? Renovate can read plain JSON, or is that different for the self_hosted?

Reply:
thanks I was a bit confused in the morning :sweat_smile: